src/cli/index.js

```diff
@@ -0,0 +1,110 @@
+import qrcode from 'qrcode-terminal';
+import { Promise } from 'es6-promise';
+import chalk from 'chalk';
+import { getInstallations as getBrowserInstallations } from 'testcafe-browser-natives';
+import CliArgumentParser from './argument-parser';
+import createTestCafe from '../';
+
+const ERR_PREFIX = chalk.red('ERROR ');
+
+function exit (code) {
+    // NOTE: give process time to flush output.
+    // It's necessary for some environments.
```

Comment:
NOTE: give a process time to flush the output.
It's necessary in some environments.